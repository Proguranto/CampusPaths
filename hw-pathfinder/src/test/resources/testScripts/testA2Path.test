# Find a path with Dijkstra's algorithm when
# the graph is acyclic and there is 2 paths
# from start to dest node.

CreateGraph graph4

AddNode graph4 S
AddNode graph4 1
AddNode graph4 2
AddNode graph4 3
AddNode graph4 D

AddEdge graph4 S 1 1.0
AddEdge graph4 S 2 4.0
AddEdge graph4 1 3 1.0
AddEdge graph4 3 D 3.14159265359
AddEdge graph4 2 D 5.0

FindPath graph4 S D
