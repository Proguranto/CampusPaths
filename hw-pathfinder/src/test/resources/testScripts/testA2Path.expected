# Find a path with Dijkstra's algorithm when
# the graph is acyclic and there is 2 paths
# from start to dest node.

created graph graph4

added node S to graph4
added node 1 to graph4
added node 2 to graph4
added node 3 to graph4
added node D to graph4

added edge 1.000 from S to 1 in graph4
added edge 4.000 from S to 2 in graph4
added edge 1.000 from 1 to 3 in graph4
added edge 3.142 from 3 to D in graph4
added edge 5.000 from 2 to D in graph4

path from S to D:
S to 1 with weight 1.000
1 to 3 with weight 1.000
3 to D with weight 3.142
total cost: 5.142
